src/components/tooltip/index.js

```diff
@@ -77,7 +80,7 @@ const Tooltip = ({
         </TooltipLabel>
       </TooltipHeader>
     ) }
-    <TooltipBody notitle={ disableTooltipTitle }>
+    <TooltipBody notitle={ disableTooltipTitle.toString() }>
```

Comment:
what's the reason for doing this?

Reply:
There was a warning in the browser about notitle being boolean. So I switch to work with it as string...maybe something with goober here? This is where I use notitle...

Reply:
perhaps goober, should just suppress. it's semantically weirder than the warning itself to convert boolean to string for string comparison with `=== 'true'`

Reply:
How do you suppress it? I noticed that if I do this 
` <TooltipBody notitle={ disableTooltipTitle.toString() }>`
the following worked as if notitle was boolean:
`margin-left: ${({ notitle }) => notitle ? '0' : '14px'};`
I guess I can try use it like this...?

Reply:
@woozyking Can you check the last version?